src/components/AppFooter/AppFooter.tsx

```diff
@@ -1,15 +1,18 @@
-import React from 'react'
+import React from 'react';
+import FooterSponsors from './FooterSponsors';
 
-const AppFooter = () => {
+const AppFooter = (): JSX.Element => {
     
     return (
         <div style={{backgroundColor: '#023874', position: "absolute", bottom: "0",
-         width: "100%", height: "3.5rem"}}>
+         width: "100%"}}>
             <h1>
                 Footer
             </h1>
+            Sponsorer og samarbeidspartnere:
```

Comment:
Should "Sponsorer og samarbeidspartnere" be put within a `<p>` tag?
Not sure what the best practice is in React, so is this is more of a question:

Reply:
I agree